spec/lib/active_outbox/generators/model_generator_spec.rb

```diff
@@ -26,6 +26,95 @@
   end
   let(:timestamp_of_migration) { DateTime.now.in_time_zone('UTC').strftime('%Y%m%d%H%M%S') }
 
+  shared_examples 'creates the correct migrations for supported adapters' do
+    context 'when it is a mysql migration' do
+      before do
+        allow(ActiveOutbox::AdapterHelper).to receive_messages(postgres?: false, mysql?: true)
+      end
+
+      let(:expected_content) do
+        <<~MIGRATION
+          class ActiveOutboxCreate#{table_name.camelize}Outboxes < ActiveRecord::Migration[#{active_record_dependency}]
+            def change
+              create_table :#{table_name}#{table_name.blank? ? '' : '_'}outboxes do |t|
+                t.string :identifier, null: false, index: { unique: true }
+                t.string :event, null: false
+                t.json :payload
```

Comment:
Is ok to allow null values here? What about adding a `default: {}`?

Reply:
ahh I remember why, it's because if it's a string or json or jsonb type depending on the db adapter it's used, it was easier to just leave it nullable